x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats_sum.csv-spec

```diff
@@ -0,0 +1,35 @@
+sumWithOverflow
+required_capability: fn_sum_overflow_handling
+FROM employees
+| LIMIT 2
+| EVAL x = languages - languages + 9223372036854775807
```

Comment:
Should we multiply `languages` by a big number? That feels less likely to get optimized away one day.

Reply:
Uhm yeah, I'll remove that cheap trick